test/box/info.result

```diff
@@ -15,7 +15,7 @@ box.info['unknown_variable']
 ---
 - null
 ...
-string.match(box.info.version, '^[1-9]') ~= nil
+string.match(box.info.version, '^[%a%s]*[1-9]') ~= nil
```

Comment:
Aren't it is better to match exact string instead of almost all?

Reply:
It is possible that distribution string could be different. For example Tarantool Enterprise For Mac. Builtin lua patterns does not give such freedom as in regexp ^(?:[a-zA-Z\s]*)(((([0-9]+)\.([0-9]+)\.([0-9]+)(?:-([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?)(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?)$)

Reply:
We don't offer anything like that.